src/main/java/core/basesyntax/service/strategy/strategyimpl/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
According to the checklist, don't keep all logic in a single package. Consider placing this class into a package that is dedicated to operation handlers to follow the project structure requirements.